lib/runtime/addon.js

```diff
@@ -157,8 +157,7 @@ export default class Addon {
     forIn(this.pkg.dependencies, (version, pkgName) => {
       let pkgMainPath = resolve.sync(pkgName, { basedir: this.dir });
       let pkgJSONPath = path.resolve(findup('package.json', { cwd: pkgMainPath }));
-      // TODO this shouldn't need to be aware of dist
-      let pkgDir = path.join(path.dirname(pkgJSONPath), 'dist');
+      let pkgDir = path.dirname(pkgJSONPath);
```

Comment:
Ahh - because the build process copies the package.json into the dist I guess, and `resolve()` looks up the `"main"` file, and so `pkgMainPath` really has the hidden assumption that the main file is located at the same directory level as the package.json - which in this case is true because of that build process. Okay, makes sense now lol